utilities_common/bgp_util.py

```diff
@@ -215,8 +215,10 @@ def get_bgp_summary_from_all_bgp_instances(af, namespace, display):
         except ValueError:
             ctx.fail("bgp summary from bgp container not in json format")
 
+        # exit cli command without printing the error message
         if key not in cmd_output_json:
-            ctx.fail("bgp summary from bgp container in invalid format")
+            click.echo("% No BGP neighbors found in VRF default")
```

Comment:
For IPV6 case This needs to be changed to:
**No IPv6 neighbor is configured**
For IPV4 case this needs to be changed to:
**No IPv4 neighbor is configured**

Reply:
updated in latest commit